docs/contributor_guidelines.rst

```diff
@@ -227,8 +227,8 @@ contributor with the right technical and social skills is entitled to
 ask. The people who have the power to grant such privileges are
 committed to do so in a transparent way as follows:
 
-#. The contributor posts a message `in the forum
-   <https://forum.securedrop.org/>`__ asking for privileges (review or
+#. The contributor posts a message `on Gitter
+   <https://app.gitter.im/#/room/#freedomofpress_securedrop:gitter.im>`__ asking for privileges (review or
```

Comment:
```suggestion
   <https://gitter.im/freedomofpress/securedrop>`__ asking for privileges (review or
```

Can we use the standard Gitter URL we use everywhere else instead of one that uses the Element URL syntax?

Reply:
Done!